internal/services/storage/storage_account_resource.go

```diff
@@ -273,10 +261,7 @@ func resourceStorageAccount() *pluginsdk.Resource {
 				Type:     pluginsdk.TypeString,
 				Optional: true,
 				Default: func() interface{} {
```

Comment:
fwiw this can become:

```suggestion
				Default: string(storage.MinimumTLSVersionTLS12),
```